### DISCUSSION TITLE
ARM or AMD?

### DISCUSSION THREAD

**Discussion by BrighamEaquinto** [quarto-dev/quarto-cli#5697] [Q&A] [answered]
I recently tried downloading quarto on an ARM chip Chromebook with out success. I did some digging and I think Quarto is only supported on the more robust AMD chips. Can anyone verify this?

**Comment by jimjam-slam** [comment, marked as answer]
I don't have a Chromebook, so I can't answer your question directly, but I can say that there are Linux downloads for both x64 and ARM 64 chips 🥳 https://quarto.org/docs/download/ Can you run the ARM 64 Linux version on a Chromebook?

**Reply by BrighamEaquinto** [reply]
James, 

Thank you for the quick and accurate reply. Either I over look that on the downloads page or it's a recently added option. 
Thank's for your help.

Brigham

![Screenshot 2023-05-26 7 23 16 PM](https://github.com/quarto-dev/quarto-cli/assets/72897055/d11e36e4-06fb-406f-a91e-e121d0937afa)
